airbyte-integrations/bases/base-normalization/dbt-project-template/macros/cross_db_utils/json_operations.sql

```diff
@@ -3,82 +3,101 @@
     - Bigquery: JSON_EXTRACT(json_string_expr, json_path_format) -> https://cloud.google.com/bigquery/docs/reference/standard-sql/json_functions
     - Snowflake: JSON_EXTRACT_PATH_TEXT( <column_identifier> , '<path_name>' ) -> https://docs.snowflake.com/en/sql-reference/functions/json_extract_path_text.html
     - Redshift: json_extract_path_text('json_string', 'path_elem' [,'path_elem'[, …] ] [, null_if_invalid ] ) -> https://docs.aws.amazon.com/redshift/latest/dg/JSON_EXTRACT_PATH_TEXT.html
+    - Postgres: json_extract_path_text(<from_json>, 'path' [, 'path' [, ...]]) -> https://www.postgresql.org/docs/12/functions-json.html
 #}
 
+{# format_path --------------------------------------------------     #}
+{% macro format_path(json_path_list) -%}
+  {{ adapter.dispatch('format_path')(json_path_list) }}
+{%- endmacro %}
+
+{% macro default__format_path(json_path_list) -%}
+  {{ '.' ~ json_path_list|join('.') }}
+{%- endmacro %}
+
+{% macro postgres__format_path(json_path_list) -%}
+ {{"'" ~ json_path_list|join("','") ~ "'" }}
+{%- endmacro %}
+
+{% macro bigquery__format_path(json_path_list) -%}
+  {{'"$' ~ json_path_list|join('"."') ~ '"'}}
+{%- endmacro %}
+
+
 {# json_extract -------------------------------------------------     #}
 
-{% macro json_extract(json_column, json_path) -%}
-  {{ adapter.dispatch('json_extract')(json_column, json_path) }}
+{% macro json_extract(json_column, json_path_list) -%}
+  {{ adapter.dispatch('json_extract')(json_column, format_path(json_path_list)) }}
```

Comment:
This is the call to the `adapter.dispatch` function, it shouldn't need to make the call to `format_path` right?

(you didn't do so on the other variants of the macro either)

Reply:
correct, I think it shouldn't need a format_path call